### PR TITLE
feat1: Add Note

### PR DESCRIPTION
**Pull request by SinchanaMS**
Added Auth and Add Note functionality.

Files to Check:
1. [Header.jsx](https://github.com/SinchanaMS/notes-app-react/pull/1/files#diff-8c468c347b974539dbc7411567675397475e1aeac509adf899831746084c1179)
2. [NoteContext.jsx](https://github.com/SinchanaMS/notes-app-react/pull/1/files#diff-a68fe59469917410a97e357369e204cb7542560e171e4982f1923c3604c7d710)
3. [Editor.jsx](https://github.com/SinchanaMS/notes-app-react/pull/1/files#diff-9b236b1937ff5f05a6b5fc3f2e8617684bf19d961b50c44529684d70651dc592)
4. [AddNote.jsx](https://github.com/SinchanaMS/notes-app-react/pull/1/files#diff-b3a015ff9dc8950fdd347da0338ca86af40a4c5979aa3ad5d4e485dda1217904)
5. [Note.jsx](https://github.com/SinchanaMS/notes-app-react/pull/1/files#diff-3c01810d057a0bb7c7f9c539fbcfac251e5cc99c6b2a15ed543ff80105a4e280)
6. [NotesList.jsx](https://github.com/SinchanaMS/notes-app-react/pull/1/files#diff-0453023648979c0dab43b2d6a9bcebbd4e7573cd4ffad227467c27bde943ce02)
7. [App.js](https://github.com/SinchanaMS/notes-app-react/pull/1/files#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67)

Preview: 

https://user-images.githubusercontent.com/94738532/161711419-1c558b05-efb3-4f1d-89c5-02c03458248c.mp4

### CODE REVIEW

File: src/components/AddNote.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    try {
+      const response = await axios.post('/api/notes', { note }, {
+        headers: {
+          authorization: localStorage.getItem("userToken")
+        }
+      })
+      if (response.status === 201) {
+       setNotes(response.data.notes)
+       setNote({...note, title: ""})
+       setNote({...note, body: ""})
```
Comment (m26unkwn) [inline src/components/AddNote.jsx:18]: why you are writing it two times. 

you could have done like this.

`setNote({...note,title:"",body:""})`

like this.

File: src/pages/Authentication/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    e.preventDefault()
+    sendLoginData(loginData)
+  }
+
+  const sendLoginData = async (loginData) =>{
+    try {
+      const response = await axios.post("/api/auth/login", loginData)
+      console.log(response)
+      if (response.status === 200){ 
+      const {data} = response;
```
Comment (m26unkwn) [inline src/pages/Authentication/Login.jsx:32]: You can destructure this at line number 29.

`const {data} = await some api`

If you are not using the `console.log()` remove it.

**Review by m26unkwn — APPROVED** [review]
You can look at the comment.

Your code structure is good and clean.

File: src/App.css [hunk 3]
```diff
@@ -1,3 +1,100 @@
-.App {
-  text-align: center;
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
+@import url('https://fonts.googleapis.com/css2?family=Aleo:ital,wght@0,300;0,400;0,700;1,400&family=Amatic+SC:wght@400;700&family=Andika+New+Basic:ital,wght@1,700&family=Comforter&family=Montserrat:ital,wght@1,800&family=Open+Sans:wght@300;500&family=Roboto:wght@100;700&display=swap');
+@import url("https://clay-ui.netlify.app/stylesheets/index.css");
```
Comment (srejitk) [inline src/App.css:3]: Move these to index.html , that will look cleaner.

**Review by srejitk — APPROVED** [review]
Looks good. Liked your color choice, and good work styling the quill editor.

File: src/pages/Authentication/Login.jsx (hunk 2 above)
Comment (SinchanaMS) [reply src/pages/Authentication/Login.jsx:32]: Thanks, I'll try that!

File: src/components/AddNote.jsx (hunk 1 above)
Comment (SinchanaMS) [reply src/components/AddNote.jsx:18]: I had tried that, but it wasn't setting both the values to empty simultaneously. I do plan on revisiting this. Thank you.

**Comment by SinchanaMS** [comment]
Thank you for the reviews!

**Pull request merged**